### PR TITLE
Remove own themebuilder linter

### PR DESCRIPTION
**Pull request by babich-a**

### CODE REVIEW

File: themebuilder-scss/src/modules/compile-manager.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import PreCompiler from './pre-compiler';
 import resolveBundle from './bundle-resolver';
 import PostCompiler from './post-compiler';
 import BootstrapExtractor from './bootstrap-extractor';
+// eslint-disable-next-line
```
Comment (San4es) [inline themebuilder-scss/src/modules/compile-manager.ts]: What rule do we want to disable here?

File: themebuilder-scss/src/modules/compiler.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import * as sass from 'sass';
+// eslint-disable-next-line
```
Comment (San4es) [inline themebuilder-scss/src/modules/compiler.ts]: What rule do we want to disable here?

File: themebuilder-scss/src/modules/post-compiler.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import CleanCSS, { Options } from 'clean-css';
 import AutoPrefix from 'autoprefixer';
 import PostCss from 'postcss';
 import commonOptions from './clean-css-options';
+// eslint-disable-next-line
```
Comment (San4es) [inline themebuilder-scss/src/modules/post-compiler.ts]: What rule do we want to disable here?

File: themebuilder-scss/src/modules/compile-manager.ts (hunk 1 above)
Comment (babich-a) [reply themebuilder-scss/src/modules/compile-manager.ts]: The module imported in the next line does not exist. It is generated while themebuilder is built.

**Review by churkin — DISMISSED** [review]

File: themebuilder-scss/src/modules/compile-manager.ts (hunk 1 above)
Comment (San4es) [reply themebuilder-scss/src/modules/compile-manager.ts]: I suggest disabling a specific rule for this line. For instance:
https://github.com/DevExpress/DevExtreme/blob/5c77f06225bd4a73cd071eb1196d2868a69d7672/build/npm-bin/bundler.js#L14

**Review by San4es — APPROVED** [review]

**Review by churkin — APPROVED** [review]

**Pull request merged**